rosbag2_transport/test/rosbag2_transport/test_record.cpp

```diff
@@ -318,27 +302,24 @@ TEST_F(RecordIntegrationTestFixture, duration_and_noncompatibility_policies_mixe
 TEST_F(RecordIntegrationTestFixture, topic_qos_overrides)
 {
   const auto num_msgs = 3;
-  test_msgs::msg::Strings strict_msg;
-  strict_msg.string_value = "strict";
-  const auto strict_topic = "/strict_topic";
+  test_msgs::msg::Strings string_msg;
```

Comment:
This whole test might not do anything anymore, since we added the subscriber-qos-adaptation feature. I think even without an override the recording gets everything here now.

Reply:
are you saying we could just remove this test altogether?

Reply:
done in e78ddbe